.gitlab-ci.yml

```diff
@@ -31,20 +30,15 @@ cache: &global_cache
   cache:
     <<: *global_cache
     policy: pull-push
-  only:
-    variables:
-      - $GITLAB_CI
-      - $CI_COMMIT_SHA
-      - $CI_COMMIT_REF_NAME
-      - $CI_JOB_ID
-      - $CI_SERVER_VERSION
-      - $CI_COMMIT_BRANCH
-      - $CI_MERGE_REQUEST_ID
-      - $PERCY_TOKEN
-      - $MYSQL_DATABASE
-      - $MYSQL_ROOT_PASSWORD
 
```

Comment:
The [Gitlab docs recommend replacing only/except conditions with rules](https://docs.gitlab.com/ee/ci/yaml/#rules), and the two are mutually exclusive.